src/main/java/io/unlogged/util/ClassTypeUtil.java

```diff
@@ -6,6 +6,9 @@
 import java.util.regex.Pattern;
 
 public class ClassTypeUtil {
+
+    private ClassTypeUtil() {}
```

Comment:
adding `throw new UnsupportedOperationException("This is a utility class and cannot be instantiated");` in these utility classes can make it more readable.
@artpar your opinion?